packages/styling/src/styles/theme.ts

```diff
@@ -14,6 +14,7 @@ let _theme: ITheme = createTheme({
   disableGlobalClassNames: false
 });
 let _onThemeChangeCallbacks: Array<(theme: ITheme) => void> = [];
+loadTheme(_theme);
```

Comment:
why do you need this call here?

Reply:
good catch, this is debug code
(there ARE bugs because we never call loadTheme() in the default case... but that's a PR for another time)